src/Application/Users/Queries/UserDTO.cs

```diff
@@ -0,0 +1,17 @@
+using Application.Common.Mappings;
+using Domain.Entities;
+
+namespace Application.Users.Queries;
+
+public class UserDTO : IMapFrom<User>
```

Comment:
Rename to UserDto to ensure PascalCasing

Reply:
okay

Reply:
also move this to src/Application/Common/Models/Dtos